pages/docs/deployment/migrate-v2-to-v3.mdx

```diff
@@ -0,0 +1,223 @@
+It behaves exactly as the `/api/public/ingestion` endpoint, but will return a 200 status code with a body of `{ id: string }` type.
+Before, the endpoint returned the created score object directly.
+This change is inline with our [API reference](https://api.reference.langfuse.com/#post-/api/public/scores) and therefore not considered breaking.
+
+#### Deprecated endpoints
+
+The following endpoints are deprecated since our V2 release.
+We continue to accept requests to these endpoints and will remove them in a future release.
+Their API behaviour changes to be asynchronous and the endpoints will only return the id of the created object.
+
+- POST /api/public/events
+- POST /api/public/generations
+- PATCH /api/public/generations
+- POST /api/public/spans
+- PATCH /api/public/spans
+- POSt /api/public/traces
+
+
+
+
+## Migration Steps
+
+We tried to make the version upgrade as seamless as possible.
+If you encounter any issues please reach out to our support team or open an issue on our GitHub repository.
+
+### Before you start the upgrade
+
+Before starting your upgrade, make sure you are familiar with the contents of [our V3 hosting guide](/docs/deployment/self-host-v3).
+In addition, we recommend that you perform a backup of your Postgres database before you start the upgrade.
+Also, ensure that you run a recent version of Langfuse, ideally a version later than v2.80.0.
+
+For a zero-downtime upgrade, we recommend that you provision new instances of the Langfuse web and worker containers
+and move your traffic after validating that the new instances are working as expected.
+
+### Upgrade Steps
+
+#### 1. Provision new infrastructure
+
+Ensure that you deploy all required storage components (Clickhouse, Redis, S3/Blob Store) and have the connection information handy.
+You can reuse your existing Postgres instance for the new deployment.
+Ensure that you also have your Postgres connection details ready.
+
+#### 2. Start new Langfuse containers.
+
+Deploy the Langfuse web and worker containers with the settings from our self-hosting guide.
+Ensure that you set the environment variables for the new storage components and the Postgres connection details.
+
+At this point, you can start to test the new Langfuse instance.
+The UI should load as expected, but there should not be any traces, observations, or scores.
+This is expected, as data is being read from Clickhouse while those elements still reside in Postgres.
+
+#### 3. Shift traffic from V2 to V3
+
+Point the traffic to the new Langfuse instance by updating your DNS records or your Loadbalancer configuration.
+All new events will be stored in Clickhouse and should appear within the UI within a few seconds of being ingested.
+
+#### 4. Wait for historic data migration to complete
+
+We have introduced background migrations as part of the migration to V3.
+Those allow the Langfuse team to schedule longer-running migrations without impacting the availability of the service.
```

Comment:
I would not use "Langfuse team" but rather mention that "Langfuse" does something as the person who wrote the migration could be on the core team of maintainers but does not need to